src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -0,0 +1,92 @@
+import { GraphQLFieldConfig } from "graphql"
+import { GraphQLInt, GraphQLObjectType } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { PartnerOfferToCollectorType } from "../partnerOfferToCollector"
+
+export const collectorSignals: GraphQLFieldConfig<any, ResolverContext> = {
+  type: new GraphQLObjectType({
+    description: "Collector signals available to the artwork",
+    name: "CollectorSignals",
+    fields: {
+      bidCount: {
+        type: GraphQLInt,
+        description: "Lot bid count",
+      },
+      lotWatcherCount: {
+        type: GraphQLInt,
+        description: "Lot watcher count",
+      },
+      partnerOffer: {
+        type: PartnerOfferToCollectorType,
+        description: "Partner offer available to collector",
+      },
+    },
+  }),
+  description: "List of collector signals",
+
+  resolve: async (artwork, {}, _ctx) => {
+    return artwork.collectorSignals
+  },
+}
+
+interface EnrichedSignals {
+  bidCount?: number
+  lotWatcherCount?: number
+  partnerOffer?: { endAt: string }
+}
+
+export const enrichArtworkWithCollectorSignals = async (artwork, ctx) => {
+  const enrichedSignals: EnrichedSignals = {}
+  artwork.collectorSignals = enrichedSignals
+
```

Comment:
This function is used in the `artwork` and `filterArtworksConnection` resolvers to populate a `collectorSignals` object. I did not make efforts to batch requests yet, and some requests might be to the same endpoints that another field would request, but it works:
<details><summary>Single artwork (auction lot) signals query</summary>
<p>

![image](https://github.com/user-attachments/assets/9fc8a5fc-7552-4317-bb83-f9508e598c64)
</p>
</details> 

<details><summary>Single artwork acquireable (Partner offer) signals query</summary>
<p>
<img width="801" alt="image" src="https://github.com/user-attachments/assets/be874860-731f-461a-a39b-80c13ff417ca">
</p>
</details> 

<details><summary>Connection query - acquirable (partner offers)</summary>
<p>

![image](https://github.com/user-attachments/assets/f5701a89-419a-4f38-b636-e2f664e15e50)
</details> 
</p>

<details><summary>Connection query - auction lots</summary>
<p>
<img width="1141" alt="image" src="https://github.com/user-attachments/assets/f8236168-ef56-4b00-a575-6ba5c7ae67bf">
</p>
</details>